### ISSUE TITLE
kestävyys

### ISSUE THREAD

**Issue by Finto-ehdotus** [Finto-ehdotus/YSE#12382]
**Käsitteen tyyppi**

Muutos olemassa olevaan käsitteeseen

**prefabel**

[kestävyys](http://www.yso.fi/onto/yso/p10119)

**Tila**

Käsittelyssä

**Ehdotettu muutos**

Kestävyys-käsitteeseen pitäisi lisätä myös merkitys kestävän kehityksen mukaisena.

**Perustelut ehdotukselle**

Tutkimuksessa käsitettä kestävyys käytetään laajasti englanninkielisen käsitteen sustainability vastineena.

**Ehdottajan organisaatio**

Jyväskylän yliopisto

**Comment by mirmaid** [comment]
Toivotaankohan tässä ohjaustermejä muilla kielillä vai jotain muuta ratkaisua?

**Comment by mariaforsen** [comment]
Sv. on hållbarhet (resistens), eli sustainability/kestävä kehitys-näkökulmaa ei saa mukaan tähän, tekemättä muitakin muutoksia käsitteen määritelmään jne.
Ks. https://termipankki.fi/tepa/fi/haku/kest%C3%A4vyys

Huom. alakäsitteet nyt:
[korroosionkestävyys](https://finto.fi/yso/fi/page/p17742)
[kuivuudenkestävyys](https://finto.fi/yso/fi/page/p20472)
[kylmänkestävyys](https://finto.fi/yso/fi/page/p3062)
[pakkasenkestävyys](https://finto.fi/yso/fi/page/p13990)
[palonkestävyys](https://finto.fi/yso/fi/page/p913)
[säteilynkestävyys](https://finto.fi/yso/fi/page/p39907)
[taudinkestävyys (kasvit)](https://finto.fi/yso/fi/page/p11869)
[tuholaiskestävyys](https://finto.fi/yso/fi/page/p26945)

Tarkenne on siis lisätty ruotsiksi erotukseksi käsitteestä hållbarhet (livsmedel)=säilyvyys
https://svenska.se/tre/?sok=h%C3%A5llbarhet&pz=1

**Comment by mirmaid** [comment]
Pelkkä _kestävyys_ sopii kyllä suomeksi myös siihen, että jokin on ekologisesti tai sosiaalisesti kestävää.

**Comment by ilona-x** [comment]
YSOssa erikseen ekologinen kestävyys & sosiaalinen kestävyys, joille voisi varmaan lisätä kestävyys-alkuiset ohjaustermit?

**Comment by mirmaid** [comment]
^ Voi lisätä, mutta en usko että se välttämättä estää käyttämästä kestävyys-käsitettä tässä kontekstissa.

**Comment by ilona-x** [comment]
Melinda-haku WSU=kestävyys AND WSU="kestävä kehitys" NOT WSU="ekologinen kestävyys" NOT WSU="sosiaalinen kestävyys" tuottaa 205 viitettä

**Comment by mariaforsen** [comment]
Myös englanniksi on nyt siis endurance (ei sustainability).

**Comment by ilona-x** [comment]
Joo, juuri tuo englannin kielen vastine on saanut minut epäilemään, että tätä termiä ei ole tarkoitus käyttää kestävän kehityksen yhteydessä, ja myös alakäsitteet viittaisivat tähän, kun ekologinen kestävyys ei ole kestävyyden alakäsite. Mutta en mene vannomaan, ettenkö olisi joskus tätä termiä lisännyt/keepannut kestävää kehitystä käsitteleviin aineistoihin, kun suomeksi se tosiaan sopisi termimuodon puolesta myös niihin.

**Comment by mirmaid** [comment]
Myöskään eng. _sustainability_ ei rajaudu mihinkään tiettyyn tarkkaan sektoriin, esim. tässä https://melinda.kansalliskirjasto.fi/byid/019219476 aiheena vapaan lähdekoodin ohjelmistojen kestävyys/sustainability. Sustainable = vakaa, pysyvä, kestävä, vakaalla pohjalla oleva.

**Comment by EsaMPKK** [comment]
Melindassa kaikki sanat urheil? + asiasana kestävyys  = 137 viitettä
Kaikki sanat kunto? + asiasana kestävyys = 209 viitettå
Kaikki sanat liiku? + asiasana kestävyys = 192 viitettä
Tarkennetussa haussa asiasanat fyysinen kunto + kestävyys = 103 viitettä
Asiasanat valmennus + kestävyys = 21 viitettä.

**Comment by mirmaid** [comment]
Jos tätä pitää lähteä jakamaan, käsitellään varmaankin vasta ongelmakokouksessa, mutta voisi katsoa alustavasti. Jako ei ehkä ole kahteen jos tehdään.

**Comment by mariaforsen** [comment]
Urheilussa termi on ruotsiksi aivan eri: uthållighet.
Fyysinen kestävyys tai kestävyys (liikunta) voisi olla erillisenä terminä? Kuten MESHissä https://finto.fi/mesh/fi/page/D010807 Vai onko se liian lähellä fyysinen kunto -termiä?

Kannatan jakoa.

Onko kestävyys "yhteiskunnallisena ominaisuutena" (oli sitten kyse ympäristöstä, taloudesta, sosiaalisesta oikeudenmukaisuudesta tms.) kuitenkin hyvin lähellä kestävä kehitys -termiä? Eli voisiko siihen ohjata?
Ks. esim.https://tieke.fi/mita-kestavyys-tarkoittaa-sinulle/

**Comment by ilona-x** [comment]
FinMeSH: fyysinen kestävyys - Physical Endurance - fysisk uthållighet http://www.yso.fi/onto/mesh/D010807

LCSH: [Physical fitness](http://id.loc.gov/authorities/subjects/sh85101540)  (linkattu YSOn fyysinen kunto)
Variants    
    Endurance, Physical
    Fitness, Physical
    Physical endurance
    Physical stamina
    Stamina, Physical

Wikidata: endurance - uthållighet https://www.wikidata.org/wiki/Q778298

**Comment by mirmaid** [comment]
Vähän mietin voisiko joku ajatella fyysisen kestävyyden soveltuvan myös rakenteille ja materiaaleille. Mutta ehkä se on estettävissä ohjaustermein, jos on erikseen termi niille. Sen tarkenne pitää ehkä kuitenkin päättää samalla kuin tämä, ettei sanastossa ole yhtäkkiä fyysinen kestävyys ja kestävyys (fysiikka) 😅

Sanastossa on myös _kestävyysharjoittelu_ ja _kestävyyslajit_. Esim. tässä https://melinda.kansalliskirjasto.fi/byid/019369822 _fyysinen kunto_ olisi sopiva, aineistossa termi on _kestävyyskunto_.

Aineistoa on myös esim. eläin- ja kasvilajeista jotka ovat kestäviä, miten ne suhteutuvat tähän?

**Comment by mariaforsen** [comment]
Sama termi ei ainakaan ruotsiksi sovi eläin- ja kasvilajien kestävyydestä (eri asia kyseessä, esim. pakkaskestävyys jne.). Lajeista käytetään ruotsiksi esim. termiä härdighet, härdig art (fi. kestävä laji), eli laji joka kestää jotain vaikeita ulkoisia olosuhteita tai vaikutuksia.

**Comment by AuHo22** [comment]
_"Vähän mietin voisiko joku ajatella fyysisen kestävyyden soveltuvan myös rakenteille ja materiaaleille."_ Melindasta löytyy viitteitä, joissa voidaan nimenomaan tarkoittaa esim. siltarakenteitten kestävyyttä, eikä niihin sovi mikään kestävyys-termin alakäsitteistä. 
Mm. 019530041 ja 019450206  resilience / robustness
